.eslintrc.json

```diff
@@ -8,6 +8,10 @@
       "func-names": 0,
       "implicit-arrow-linebreak": 0,
       "no-use-before-define": ["error", { "variables": false }],
-      "react/prop-types": ["error", { "ignore": ["navigation"] }]
+      "react/prop-types": ["error", { "ignore": ["navigation"] }],
+      "object-curly-newline": 0
+    },
+    "globals": {
+        "fetch": true
```

Comment:
Why is this here?

Reply:
The linter doesn't recognize "fetch" as a node or J's object. So I have to declare it as such even though react knows what's going on